src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -10,6 +9,7 @@ import linkConfig from '@/mixins/linkConfig';
 import get from 'lodash/get';
 import { id as laneId } from '../poolLane';
 import { expressionPosition } from './sequenceFlowConfig';
+import  inclusiveGateway from '../inclusiveGateway/index';
```

Comment:
For the other components, the ID is specified as a non-default export. I think this should be changed to that as well, to keep it consistent:

```
import { id as inclusiveGatewayId } from '../inclusiveGateway/index';
```

This also lets users know, from the imports, that only the ID is required and used, and not the entire config.